### PR TITLE
ac:image macro

### PR DESCRIPTION
**Pull request by Fethbita**
This PR is created for the issue #245. Feel free to edit or change commits.

### CODE REVIEW

**Review by mrueg — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -494,6 +494,22 @@ This is a [link to an existing confluence page](ac:Pagetitle)
...
+Use the following macro:
+```markdown
+<!-- Macro: \!\[.*\]\((.+)\)\<\!\-\- (.*) \-\-\>
+     Template: ac:image
+     Attachment: ${1}
+     Width: ${2} -->
+```
+And attach any image with the following
+```markdown
+![Example](../images/example.png)<!-- 300 -->
```
Comment (kovetskiy) [inline README.md:508]: Nice idea!

I'd make it more explicit, i.e. `<!-- width=300 -->` so in the future we will be able to add `height=` as well, but I it's ok to leave it like this for the sake of example

Comment (Fethbita) [reply README.md:508]: Yes, that sounds great. I allowed the maintainers to add new commits to my branch so if you'd like, please add it as a commit.
` <!-- Macro: \!\[.*\]\((.+)\)\<\!\-\- (.*) \-\-\> ` turns to ` <!-- Macro: \!\[.*\]\((.+)\)\<\!\-\- width=(.*) \-\-\> ` and `Example<!-- 300 -->` turns to `Example<!-- width=300 -->`.

**Comment by joyfulrabbit** [comment]
I'd really like to use this!

File: README.md (hunk 1 above)
Comment (Fethbita) [reply README.md]: Added.

**Pull request merged**